src/include/local/stir/motion/NonRigidObjectTransformationUsingBSplines.h

```diff
@@ -53,15 +53,25 @@ class NonRigidObjectTransformationUsingBSplines
 public:
   static const char * const registered_name;
 
+  // Default constructor
   NonRigidObjectTransformationUsingBSplines();
 
+  // Give x, y and z components of the deformation field images separately
+  NonRigidObjectTransformationUsingBSplines(const std::string, const std::string, const std::string, int &bspline_order);
```

Comment:
`const std::string&`. Really no reason to use `int& bspline_order` as that tells the user that this parameter can be modified.
Also next constructor.